mrtt-ui/src/library/useInitializeQuestionMappedForm.js

```diff
@@ -30,6 +30,7 @@ const useInitializeQuestionMappedForm = ({
             }
           })
           .catch(() => {
+            setIsLoading(false)
```

Comment:
should be set to false if we have an error message, but realized we don't really have an error page/state at the moment. should add it to the todos

Reply:
I just left it loading for that reason. This works too